src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2252,6 +2245,16 @@ namespace Akka.Configuration.Hocon
 }
 namespace Akka.Dispatch
 {
+    public sealed class ActionRunnable : Akka.Dispatch.IRunnable
```

Comment:
The `IRunnable` classes are used by the `MessageDispatcher` to run tasks - they may not stick around depending on how performance review goes. The idea is to avoid constant delegate allocations by using classes that can encapsulate all of the state needed to execute an asynchronous action internally, and then use a static delegate to execute those. This is similar to how the task queues work in DotNetty and other parts of the TPL.